src/pages/developers/index.tsx

```diff
@@ -0,0 +1,470 @@
+import { Image } from "@/components/Image"
+import DogeImage from '@/public/doge-computer.png'
+import HeroImage from "@/public/heroes/developers-hub-hero.jpg"
+import FeedbackCard from "@/components/FeedbackCard"
+import HubHero from "@/components/Hero/HubHero"
+
+const Page = (props: ChildOnlyProp) => (
+  <Flex
+    flexDirection="column"
+    alignItems="center"
+    w="full"
+    my={0}
+    mx="auto"
+    {...props}
+  />
+)
+
+const GrayContainer = (props: ChildOnlyProp) => (
+  <Box
+    w="full"
+    py={16}
+    px={0}
+    mt={8}
+    bg="grayBackground"
+    boxShadow="inset 0px 1px 0px var(--eth-colors-tableItemBoxShadow)"
+    {...props}
+  />
+)
+
+const Content = (props: ChildOnlyProp) => (
+  <Box py={4} px={8} w="full" {...props} />
+)
+
+const Subtitle = (props: TextProps) => (
+  <Text fontSize="xl" lineHeight="140%" color="text200" {...props} />
+)
+
+const MonoSubtitle = (props: ChildOnlyProp) => <OldHeading mb={0} {...props} />
+
+const StyledCardContainer = (props: ChildOnlyProp) => (
+  <SimpleGrid columns={[1, 1, 2, 4]} mx={-4} mt={8} mb={12} {...props} />
+)
+
+const TwoColumnContent = (props: ChildOnlyProp) => (
+  <Flex
+    justifyContent="space-between"
+    alignItems={{ base: "flex-start", lg: "center" }}
+    flexDirection={{ base: "column", lg: "row" }}
+    w="100%"
+    {...props}
+  />
+)
+
+const ThreeColumnContent = (props: ChildOnlyProp) => (
+  <Flex
+    py={0}
+    px={8}
+    w="full"
+    justifyContent="space-between"
+    alignItems={{ base: "flex-start", lg: "flex-start" }}
+    flexDirection={{ base: "column", lg: "row" }}
+    {...props}
+  />
+)
+
+const Column = (props: ChildOnlyProp) => (
+  <Box flex="1 1 33%" mb={6} mr={8} w="full" {...props} />
+)
+const RightColumn = (props: ChildOnlyProp) => (
+  <Box flex="1 1 33%" mb={6} mr={0} w="full" {...props} />
+)
+const IntroColumn = (props: ChildOnlyProp) => (
+  <Box
+    flex="1 1 33%"
+    mb={6}
+    mt={{ base: 0, lg: 32 }}
+    mr={{ base: 0, sm: 8 }}
+    w="full"
+    {...props}
+  />
+)
+
+const StyledCard = (props: ICardProps) => {
+  const tableBoxShadow = useColorModeValue("tableBox.light", "tableBox.dark")
+
+  return (
+    <Card
+      boxShadow={tableBoxShadow}
+      m={4}
+      p={6}
+      {...props}
+      _hover={{
+        borderRadius: "4px",
+        boxShadow: "0px 8px 17px rgba(0, 0, 0, 0.15)",
+        background: "tableBackgroundHover",
+        transition: "transform 0.1s",
+        transform: "scale(1.02)",
+      }}
+    />
+  )
+}
+
+const StyledCallout = chakra(Callout, {
+  baseStyle: {
+    flex: { base: "auto", md: "1 1 416px" },
+  },
+})
+
+export const getStaticProps: GetStaticProps = async (
+  context
+) => {
+  const { locale } = context
+  // load i18n required namespaces for the given page
+  const requiredNamespaces = getRequiredNamespacesForPath('/developers')
+
+  return {
+    props: {
+      ...(await serverSideTranslations(locale!, requiredNamespaces)),
+    },
+  }
```

Comment:
To display the last deploy date we need to pass it as a prop

```suggestion
  const requiredNamespaces = getRequiredNamespacesForPath('/developers')
  const lastDeployDate = getLastDeployDate()

  return {
    props: {
      ...(await serverSideTranslations(locale!, requiredNamespaces)),
      lastDeployDate,
    },
  }
```

Reply:
To avoid this:
![image](https://github.com/ethereum/ethereum-org-fork/assets/468158/153194ca-54f8-49bd-8083-206701ced35c)